tests/test_torch_simple.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+
+import torch
+import alpa.torch.optim as torchoptim
```

Comment:
Nit: replace by `atorchoptim`?

Reply:
`torchoptim` technically doesn't have alpa-related topic inside (I was thinking about moving it into a standalone repository, but putting it in alpa for now for easier versioning/CI testing). And I also want to avoid the `atorch` vs. `torch` question :)